runtime/platform/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 cmake_minimum_required(VERSION 3.5.1 FATAL_ERROR)
 include($ENV{TAPASCO_HOME_RUNTIME}/cmake/Tapasco.cmake NO_POLICY_SCOPE)
+
+set(CMAKE_C_FLAGS_RELEASE "-DNDEBUG -O2")
```

Comment:
Please use the target based commands and don't randomly replace variables (which might contain other information).

Reply:
e.g. ` target_compile_definitions(platform PRIVATE -DNDEBUG)`

Reply:
Additional comment: -O2 is only necessary for the ARM64 Platforms, right? If so, please wrap the O2 in a condition that leaves O3 in place for all other platforms.

Reply:
Fix added in aead633